articles/api/authentication/_login.md

```diff
@@ -229,59 +195,41 @@ GET https://${account.namespace}/authorize?
 ```
 
 ```javascript
-<script src="${auth0js_url}"></script>
+// Script uses auth0.js v8. See Remarks for details.
```

Comment:
@jeffreylees This auth0.js script is not valid, it has username/password while the passive authentication is just a redirect to the Auth0 Login Page that will show the Login Widget where the user can login manually with email and password. 

I think we need something like the [buildAuthorizeUrl](https://github.com/auth0/auth0.js#auth0authentication). Can you please check with brucke and find the correct script for this case? Maybe we should add this to the auth0.js index as well.